tools/ya_getopt/ya_getopt.c

```diff
@@ -0,0 +1,318 @@
+/* -*- indent-tabs-mode: nil -*-
```

Comment:
Can you add the library as a submodule instead of as a copy?

Reply:
Yes, I will do that later today